.github/workflows/update-oam-defs.yml

```diff
@@ -88,11 +88,7 @@ jobs:
             apiGroup=$(./kubeopenapi-jsonschema --location ~/linkerd.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..spec' --o-filter "$[].group" -o json | jq '.[]' | tr -d '"')
 
             ./kubeopenapi-jsonschema --location ~/linkerd.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..openAPIV3Schema.properties.spec' --o-filter "$[]" -o json |\
-            jq '
-            .[]
-            | .version = "'$NEW_LINKERD_VERSION'"
-            | ."object-type" = "'$nameUpper'"
-            | ."service-mesh" = "'$meshName'" ' > $dirPath/$nameLower.linkerd.meshery.layer5io.schema.json
+            jq '.[] | ."title" = "'$nameLower'"'  > $dirPath/$nameLower.linkerd.meshery.layer5io.schema.json
```

Comment:
Is this now consistent with the approach is all the rest of the adapters?

Reply:
oops , these came along as well. Yeah to make it consistent, just need to make a slight change